pandas/tests/interchange/test_impl.py

```diff
@@ -278,7 +278,7 @@ def test_empty_pyarrow(data):
     expected = pd.DataFrame(data)
     arrow_df = pa_from_dataframe(expected)
     result = from_dataframe(arrow_df)
-    tm.assert_frame_equal(result, expected)
+    tm.assert_frame_equal(result, expected, check_column_type=False)
```

Comment:
The difference is
```python
(Pdb) p result.columns
Index([], dtype='object')
(Pdb) p expected.columns
RangeIndex(start=0, stop=0, step=1)
```
Given that this only affects dataframes with zero columns, are we OK with this difference? I am

Reply:
Ideally `RangeIndex` would be better, but OK to punt